client/src/components/ProfileCard.tsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import Card from '@material-ui/core/Card';
+import CardMedia from '@material-ui/core/CardMedia';
+import CardContent from '@material-ui/core/CardContent';
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+import Avatar from '@material-ui/core/Avatar';
+import Grid from '@material-ui/core/Grid';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+
+interface ProfileCardProps {
+  image?: string;
+  username?: string;
+  age?: number;
+  distance?: number;
+  occupation?: string;
+  company?: string;
+}
+
+interface GithubCardProps {
+  image?: string;
+  username?: string;
+  commits?: number;
+  projects?: number;
+}
+
+interface CardProps {
+  profileData?: ProfileCardProps;
+  githubData?: GithubCardProps;
+}
+
+const useStyles = makeStyles({
+  overlay: {
+    width: '100%',
+    height: '100%',
+    padding: 0,
+    '&:last-child': {
+      paddingBottom: 0,
+    },
+    color: 'white',
+    textShadow: '1px 1px 1px black',
+  },
+  profilePhoto: {
+    height: '100vh',
+    maxHeight: '800px',
+  },
+  paper: {
+    maxWidth: '640px',
+    maxHeight: '800px',
+  },
+  avatar: {
+    width: 60,
+    height: 60,
+    marginRight: '0.5rem',
+  },
+  profileName: {
+    fontWeight: 500,
+    fontSize: '1.2rem',
```

Comment:
```suggestion
    fontSize: '1.5rem',
```
Making the profile name a bit bigger looks better imo, looks good otherwise!

Reply:
Sure!